micro_sam/sam_annotator/_widgets.py

```diff
@@ -131,12 +131,51 @@ def label_changed(new_label):
 
     return label_widget
 
+def _process_tiling_inputs(tile_shape_x, tile_shape_y, halo_x, halo_y):
+    tile_shape = (tile_shape_x, tile_shape_y)
+    halo = (halo_x, halo_y)
+    # check if tile_shape/halo are not set: (0, 0)
+    if isinstance(tile_shape, tuple):
+        if all(item == 0 for item in tile_shape):
+            tile_shape = None
+        # check if at least 1 param is given
+        elif tile_shape[0] != 0 or tile_shape[1] != 0:
+            max_val = max(tile_shape[0], tile_shape[1])
+            if max_val < 256: # at least tile shape >256
+                max_val = 256
+            tile_shape = (max_val, max_val)
+        # if both inputs given, check if smaller than 256
+        elif tile_shape[0] != 0 and tile_shape[1] != 0:
```

Comment:
Sadly, I had to settle for this solution to set a minium value for tile_shape, because limiting the range to desired values broke the input mask resulting in unwanted values in tile_shape

Reply:
That's ok. However, we will never run into this condition, because the `or` one above will also be triggered by it.